hostapd: add page (#4211)
tldr-pages/tldr: pages/common/hostapd.md

# hostapd

> > Create an access point out of wireless interface.

Reviewer: Perhaps this will read better?
```suggestion
> Start an access point using a wireless interface.
> More information: <https://w1.fi/hostapd/>.
```

Author: Hi. Yes, that is better.

> - Create an access point:

Reviewer: Perhaps this will read better?
```suggestion
- Start an access point:
```

`sudo hostapd {{path/to/hostapd.conf}}`

> - Create an access point and run in as daemon:

Reviewer: Perhaps this will read better?
```suggestion
- Start an access point, forking into the background:
```
